tests/marqo_test.py

```diff
@@ -209,10 +209,17 @@ def setUpClass(cls) -> None:
         cls.unstructured_no_model_index_name = "unstructured_no_model_index"
         cls.structured_image_index_name_simple_preprocessing_method = \
             "structured_image_index_simple_preprocessing_method"
+        cls.unstructured_languagebind_index_name = "unstructured_languagebind_index"
```

Comment:
Since we opted to only have structured, this line might be unnecessary

Reply:
done